src/atwho.js

```diff
@@ -56,7 +52,7 @@ define([
                     return choice.name && !_.isUndefined(choice.displayLabel);
                 })
                 .value();
-    }, 500);
+    }, 0);
```

Comment:
What's the practical outcome of this change? Does it mean this will be recalculated every time `cachedMugData` is called? If yes, can the `timed` decorator/wrapper be removed?